File: benchmark/results/python-multilang/run-002/b-baseline/output/metaengine_demo/catalog/value_objects/product_description.py
"""ProductDescription value object for the catalog domain."""

from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class ProductDescription:
    """Immutable descriptive metadata for a product."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str

File: benchmark/results/ts-monolith-sonnet-inline/run-002/b-baseline/output/src/catalog/value-objects/ProductDescription.ts
import { Id } from '../../shared/value-objects/Id';
import { Timestamp } from '../../shared/value-objects/Timestamp';

/** Descriptive metadata for a product. */
export interface ProductDescription {
  id: Id;
  createdAt: Timestamp;
  updatedAt: Timestamp;
  name: string;
  description: string;
}
